.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,3 +20,4 @@
 - [ ] My code follows the accessibility standards. <!-- Guidelines: https://make.wordpress.org/core/handbook/best-practices/coding-standards/accessibility-coding-standards/ -->
 - [ ] My code has proper inline documentation. <!-- Guidelines: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/ -->
 - [ ] I've included developer documentation if appropriate. <!-- Handbook: https://wordpress.org/gutenberg/handbook/designers-developers/ -->
+- [ ] I've updated all `.native.js` files affected by refactorings/renamings (if any) in this PR.
```

Comment:
How about `.android.js` and `.ios.js`? Should be there another document explaining it in-depth? I think it might be confusing for new contributors in the current shape.

Reply:
Makes sense to include the platform specific (`.android.js`, `.ios.js`) modules too, good call.

What we want to accomplish here is to make sure people notice and ideally update the mobile files that use functions/components from the web-side, especially on trivial cases like when renaming a function or moving it into a different package.

Highly manual a process currently, but until we get better at automating those, we need a stopgap solution.

Where do you think we can put that explanation?

Reply:
A new page referenced from this file: https://github.com/WordPress/gutenberg/blob/master/docs/contributors/develop.md might help to raise awareness that this project works with React Native as well.

Reply:
Done with 45cacd0. Let me know how that looks to you @gziolo , thanks!